src/modules/fw_att_control/fw_att_control_params.c

```diff
@@ -735,3 +727,57 @@ PARAM_DEFINE_FLOAT(FW_DTRIM_R_FLPS, 0.0f);
  * @increment 0.01
  */
 PARAM_DEFINE_FLOAT(FW_DTRIM_P_FLPS, 0.0f);
+
+/**
+ * Pitch trim increment for spoiler configuration
+ *
+ * This increment is added to the pitch trim whenever spoilers are fully deployed.
+ *
+ * @group FW Attitude Control
+ * @min -0.25
+ * @max 0.25
+ * @decimal 2
+ * @increment 0.01
+ */
+PARAM_DEFINE_FLOAT(FW_DTRIM_P_SPOIL, 0.f);
+
+/**
+ * Spoiler landing setting
+ *
+ * Also applies to spoilerons (ailerons deflected down) if enabled in the mixer/allocation.
+ *
+ * @unit norm
+ * @min 0.0
+ * @max 1.0
+ * @decimal 2
+ * @increment 0.01
+ * @group FW Attitude Control
+ */
+PARAM_DEFINE_FLOAT(FW_SPOILERS_LND, 0.f);
+
+/**
+ * Spoiler descend setting
+ *
+ * Also applies to spoilerons (ailerons deflected down) if enabled in the mixer/allocation.
+ *
+ * @unit norm
+ * @min 0.0
+ * @max 1.0
+ * @decimal 2
+ * @increment 0.01
+ * @group FW Attitude Control
+ */
+PARAM_DEFINE_FLOAT(FW_SPOILERS_DESC, 0.f);
+
+/**
+ * Spoiler input in manual flight
+ *
+ * Chose source for manual setting of spoilers in manual flight modes.
+ * Also applies to spoilerons (ailerons deflected down) if enabled in the mixer/allocation.
+ *
+ * @value 0 Disabled
+ * @value 0 Flaps channel
+ * @value 0 Aux1
```

Comment:
these are all 0 -- supposed to be 0,1,2, right?

Reply:
https://github.com/PX4/PX4-Autopilot/pull/19329/commits/98ea2aeb1c80f716c5335c194fbe02116df85ca6